frontend/src/scenes/popup/CreatePopupPage.jsx

```
@@ -1,140 +1,320 @@
+import { useLocation, useNavigate } from "react-router-dom";
+import {
+  addPopup,
+  editPopup,
+  getPopupById,
+} from "../../services/popupServices";
+import GuideTemplate from "../../templates/GuideTemplate/GuideTemplate";
+import { emitToastError } from "../../utils/guideHelper";
+import toastEmitter, { TOAST_EMITTER_KEY } from "../../utils/toastEmitter";
+import PopupAppearance from "./PopupPageComponents/PopupAppearance/PopupAppearance";
+import PopupContent from "./PopupPageComponents/PopupContent/PopupContent";
+import RichTextEditor from "@components/RichTextEditor/RichTextEditor";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'RichTextEditor'. Consider to delete it or rename it.

'RichTextEditor' is defined here:


(lint/suspicious/noRedeclare)